Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -286,4 +287,30 @@ public override string ToString()
             return "<Teacher>{ firstName: " + firstName + ", lastName: " + lastName + ", department: " + department + ", birthDate: " + birthDate + " }";
         }
     }
+    
+    public class UpperCaseStringIonSerializer : IonSerializer<string>
```

Comment:
Does not belong in `TestObjects.cs`

Reply:
Why not? These serializers are just meant for testing the custom serializer functionality.